demo/App.js

```diff
@@ -6,6 +6,7 @@
  *
  */
 
+import 'react-native-gesture-handler';
```

Comment:
Checkout the module `src/gesture-handler` which does that same import, but does have an escape hatch for web. This reminds me we should test the demo on web too.